src/tests/ffi.cpp

```diff
@@ -6744,4 +6746,108 @@ TEST_F(rnp_tests, test_ffi_import_signatures)
     assert_false(revoked);
     assert_rnp_success(rnp_key_handle_destroy(key_handle));
     assert_rnp_success(rnp_ffi_destroy(ffi));
-}
\ No newline at end of file
+}
+
+TEST_F(rnp_tests, test_ffi_export_revocation)
+{
+    rnp_ffi_t   ffi = NULL;
+    rnp_input_t input = NULL;
+
+    assert_rnp_success(rnp_ffi_create(&ffi, "GPG", "GPG"));
+    assert_rnp_success(rnp_input_from_path(&input, "data/test_key_validity/alice-sec.asc"));
+    assert_rnp_success(rnp_import_keys(ffi, input, RNP_LOAD_SAVE_SECRET_KEYS, NULL));
+    assert_rnp_success(rnp_input_destroy(input));
+
+    rnp_key_handle_t key_handle = NULL;
+    assert_rnp_success(rnp_locate_key(ffi, "userid", "Alice <alice@rnp>", &key_handle));
+    rnp_output_t output = NULL;
+    assert_rnp_success(rnp_output_to_path(&output, "alice-revocation.pgp"));
```

Comment:
I think it's better to do `rnp_output_to_null` here

Reply:
@rrrooommmaaa Fixed.